src/components/links/link/link.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import Styled from '@emotion/styled';
+import getPassThroughProps from '../../../utils/get-pass-through-props';
+import vars from '../../../../materials/custom-properties';
+
+const createStyledComponent = component => Styled(component)`
+  font-family: ${vars.fontFamily};
+  color: ${vars.colorPrimary};
+  font-size: ${vars.fontSizeDefault};
+
+  &:hover, &:focus, &:active {
```

Comment:
Just curious: no specific colour for the `:visited` state?

Reply:
yeah we will need to ask design 🤔

Reply:
Design said for the current / ongoing work we don't need visited state.